src/app/calendar/page.tsx

```diff
@@ -1,14 +1,17 @@
 import { Metadata } from "next";
 import React from "react";
 import ParcelCalendar from "@/app/calendar/ParcelCalendar";
-import getParcelCalendarEvents from "@/app/calendar/getParcelCalendarEvents";
+import {
+    getParcelsWithCollectionDate,
+    parcelWithClientName,
+} from "@/app/calendar/parcelCalendarFunctions";
 
 const CalendarPage = async (): Promise<React.ReactElement> => {
-    const events = await getParcelCalendarEvents();
+    const parcelsWithCollectionDate: parcelWithClientName[] = await getParcelsWithCollectionDate();
```

Comment:
You might not need to specify the `parcelWithClientName[]` type :)

Reply:
Done